scripts/qa_crowbarsetup.sh

```diff
@@ -2303,6 +2355,15 @@ function custom_configuration
             if iscloudver 7plus ; then
                 proposal_set_value nova default "['attributes']['nova']['metadata']['vendordata']['json']" "'{\"custom-key\": \"custom-value\"}'"
             fi
+
+            if iscloudver 7plus && [[ $want_ironic = 1 ]] ; then
+                if [[ $hacloud = 1 ]] ; then
+                    echo "HA is not supported for ironic, not adding nova-compute-ironic role in nova"
```

Comment:
That's not really true. HA for non-ironic control plane is supported with ironic. `nova-compute-ironic` should got to a dedicated "compute" node if available. Otherwise it could be placed on one of the cluster members but I haven't tested such setups.

Reply:
Okay, I'll have to play with it a bit to see which configuration works.

Reply:
Fixed, putting it on an individual member of a cluster seems to work